SignalServiceKit/protobuf/Provisioning.proto

```diff
@@ -8,7 +8,7 @@
 // the legacy proto format.
 syntax = "proto2";
 
-package ProvisioningProto;
+package ProvisioningProtos;
```

Comment:
Our other schemas all use a plural package name, so I changed to this be consistent.